gremlin-server/src/test/java/org/apache/tinkerpop/gremlin/server/GremlinServerAuthzIntegrateTest.java

```diff
@@ -158,6 +158,7 @@ public void shouldAuthorizeGremlinLangRequestWithLambda() {
         }
     }
 
+    @Ignore("not implemented for string requests")
```

Comment:
you could probably wholly remove the "lambda" test since GLVs can't send those anymore. as mentioned elsewhere in the PR, it would be good to get a JIRA up for tracking these things and have it referenced here in the ignores so it doesn't get lost.